src/api/app/models/obs_factory/distribution_strategy_sle15.rb

```diff
@@ -16,14 +16,25 @@ def test_dvd_prefix
       '000product:SLES-cd-DVD'
     end
 
+    def sp_version(name)
+      if name.start_with? 'SUSE:SLE-15:GA'
+        nil
+      else
+        match = name.match(/SUSE:SLE-15-(.*):GA/)
+        match[1]
+      end
```

Comment:
Nitpick: This could be
```ruby
      unless name.start_with? 'SUSE:SLE-15:GA'
        match = name.match(/SUSE:SLE-15-(.*):GA/)
        match[1]
      end
```

Reply:
or
```ruby
return $1 if name.match(/SUSE:SLE-15-(.*):GA/)
```

but I guess Ana won't like that^^

Reply:
I'll refactor when I have a way to test this :)

Reply:
Fair enough